apps/web/pages/index.tsx

```diff
@@ -82,3 +93,12 @@ export default function Home() {
     </>
   )
 }
+
+
+export async function getServerSideProps(): Promise<{props: Props}> {
```

Comment:
It's not a big deal but I feel like this might not be necessary. Have you thought about maybe making a hook like `useIsFaucetEmpty` or something? Then you wouldn't have to plumb `isOutOfCelo` through the components.

Reply:
👨‍🔧